.changeset/twenty-tips-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/ui': patch
```

Comment:
Not seeing any changes to the ui package in this PR. We don't include a changeset for docs only changes, please remove this file.